### DISCUSSION TITLE
Battery charge limit not being enforced

### DISCUSSION THREAD

**Discussion by satvikmaurya** [seerge/g-helper#230] [Q&A]
I recently uninstalled MyAsus and AC to shift to G-helper and I've noticed that my limit on the battery charge level (80%) is not being honored, the laptop ends up charging to 100% after leaving it plugged in for a while. I have the 2020 G14.

Edit: I think g-helper stops enforcing the charge limit after every system restart until I manually adjust the slider. Is there a way to fix this?

**Comment by seerge** [comment]
@satvikmaurya hello. I think this should be posted https://github.com/seerge/g-helper/issues and please include a log from ``%appdata%\ghelper`` as issues instruction explains :) 

P.S. And based on what you wrote so far - it could be that asus service overrides app setting. As app explicitly sets battery limit on each start.
